shared/vehicles.lua

```diff
@@ -3,15 +3,15 @@ QBShared.VehicleHashes = {}
 
 QBShared.Vehicles = {
     --- Compacts
-    ['asbo'] = {
-        ['name'] = 'Asbo',
-        ['brand'] = 'Maxwell',
-        ['model'] = 'asbo',
-        ['price'] = 4000,
-        ['category'] = 'compacts',
-        ['categoryLabel'] = 'Compacts',
-        ['hash'] = `asbo`,
-        ['shop'] = {'pdm'},
+    ['asbo'] = { -- This is to match the spawn code of the vehicle
+        ['name'] = 'Asbo', -- this is a custom name of the car
+        ['brand'] = 'Maxwell', -- This is the vehicle's brand (ford, chevy, bmw, mercedes, etc)
+        ['model'] = 'asbo', -- this must match the spawn code of the vehicle
+        ['price'] = 4000, -- sales price of vehicle
+        ['category'] = 'compacts', -- this must match vehicleclass in vehicles.meta for the vehicle and must match with https://docs.fivem.net/natives/?_0x29439776AAA00A62
+        ['categoryLabel'] = 'Compacts', -- Customize for your VehicleShop headings (can even include spaces)
+        ['hash'] = `asbo`, -- must contain back ticks instead of single quotes and must match the spawn code of the vehicle
+        ['shop'] = {'pdm'}, -- can be a single shop or multiple shops. For multiple shops for example {'shopname1','shopname2','shopname3'}
```

Comment:
```suggestion
    ['asbo'] = { -- This has to match the spawn code of the vehicle
        ['name'] = 'Asbo', -- This is the display name of the vehicle
        ['brand'] = 'Maxwell', -- This is the vehicle's brand (ford, chevrolet, bmw, mercedes, etc)
        ['model'] = 'asbo', -- This must match the spawn code of the vehicle
        ['price'] = 4000, -- The price that the vehicle sells for
        ['category'] = 'compacts', -- This must match "VehicleClass" in vehicles.meta for the vehicle and must also exist in https://docs.fivem.net/natives/?_0x29439776AAA00A62
        ['categoryLabel'] = 'Compacts', -- Customize for your Vehicle Shop headings (can even include spaces)
        ['hash'] = `asbo`, -- The hash key of the vehicle. Either backticks or GetHashKey can be used for generating this.
        ['shop'] = {'pdm'}, -- Can be a single shop or multiple shops. For multiple shops for example {'shopname1','shopname2','shopname3'}
```

Looks great, although I think these explenations would fit better! :)

Reply:
This part:
The hash key of the vehicle. Either backticks or GetHashKey can be used for generating this.
the people don't knwo what the hell GetHashKey is they aren't programmers they just want to get a server going.
Make it simple
This has to match the spawn code of the vehicle and must be surrounded by backticks. Example of a Backtick: `